api/src/test/java/org/pmiops/workbench/api/ConceptSetsControllerTest.java

```diff
@@ -667,7 +667,18 @@ public void getConceptSetsInWorkspaceOne() {
             .getItems();
 
     assertThat(response.size()).isEqualTo(1);
-    assertThat(response).contains(defaultConceptSet);
+    assertConceptSet(response.get(0), defaultConceptSet);
```

Comment:
Had to update test cases since response no longer contains hydrated list of concepts